.eslintrc.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
       }
     }],
     'one-var': ['error', 'never'],
+    'import/no-extraneous-dependencies': ['error', {
+      /* the following folders can import devDependencies */
+      devDependencies: ['test/**', 'tests/**', 'src/debug/**', 'examples/**'],
```

Comment:
Because `src/` contains code that use ES6 imports **and** rely on some Webpack configuration (for modules resolution for example), it should be in `.npmignore` (and it is, so we're fine).  
Then, AFAIK, `dist/` would be included in the package because it's not ignored (and it is generated by `npm build` during any "NPM release process", so there shouldn't be cases where it's not there).

What I had in mind at some point was to run Babel on `src/` to output ES5+commonjs to, say, `lib/`, then we could change our `main` to `lib/Main.js` (for people using iTowns through NPM and with Browserify, Rollup, or Webpack; so their transformations/bundling/optimizations can act on the real sources rather than the webpack-generated bundle). IIRC, this is not possible today mostly (or entirely) due to the module paths in `import` statements being relative to `src/` rather than relative to the script that imports them, and the importing of `.glsl` files (we'd have to use `require('fs').readFileSync(…)` instead, and use a BRFS or gslify transform during the Babel transpilation, or require that everyone does that BRFS or gslify transformation on their side; see https://gist.github.com/mattdesl/aaf759da84cc44c22305).